docker/mariner/Dockerfile.temurin-8-jdk

```
@@ -0,0 +1,28 @@
+ARG IMAGE=cblmariner.azurecr.io/base/core
```

To secure the supply chain, we need to remove the image and the tag.
```suggestion
ARG IMAGE
```

---

Let's try with the proper golden image.